lib/galaxy/dependencies/pipfiles/default/Pipfile

```diff
@@ -87,7 +87,7 @@ cloudbridge = "*"
 social-auth-core = {version = "==3.3.0", extras = ['openidconnect']}
 cloudauthz = "==0.6.0"
 gxformat2 = "*"
-refgenconf = ">=0.7.0"
+refgenconf = ">=0.7.0, <0.10.0"  # https://github.com/galaxyproject/galaxy/issues/11601
```

Comment:
:+1:

Reply:
Can you please do the same also in `packages/app/requirements.txt` ?

Reply:
is that the same syntax ?

Reply:
I mean, as done below for sqlalchemy.